demo/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "demo-3",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@ant-design/icons": "^5.4.0",
```

Comment:
WDYT about creating a tech debt ticket for removing ant-design (if one hasn't been created already!)?

Reply:
@guillaumegaluz I know that you had some thoughts about designs changing, were you planning on continuing to use `ant-design`?